### PR TITLE
feat: add github actions for running tests

### PR DESCRIPTION
**Pull request by karlknowit**
## What was solved
Added a github action to run jest unit tests

## How was it solved
Solved it by adding a github action which runs jest from root of the repository. Only added tests will be run and services/files without tests will not.

## Why was it solved in this way
This mimics the way we are doing it in other repositories. 

## How was it tested
Tested by adding the github actions and by creating a failing test. Checked the github page if the check was successful or not. When I saw the check were failing due to the failing test, I then fixed the tests and all checks passed.

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1f391c7 Skapa github actions för att köra jest unittest i helsingborg-io-sls-api repot](https://app.clickup.com/t/1f391c7)

**Review by sepehrnoor — APPROVED** [review]
Looks good to me.

File: .github/workflows/smoke-tests.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          # note: uses a separate (dev) cache, different from the build workflow
+          key: ${{ runner.os }}-yarn-dev-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-dev-
+
+      # inject npmrc
```
Comment (yo-l1982) [inline .github/workflows/smoke-tests.yml:37]: I think these above the "- name" comments is a bit unnecessary as it is basically repeated in the "- name" yaml property.

**Review by yo-l1982 — APPROVED** [review]
Looks fine.

**Review by yo-l1982 — APPROVED** [review]

**Review by sepehrnoor — APPROVED** [review]

**Review by KenthLjungHBG — APPROVED** [review]
Nice!

**Review by dannil76 — APPROVED** [review]

**Pull request merged**